src/main.rs

```diff
@@ -312,6 +313,17 @@ enum UtilCommand {
         #[arg(short, long)]
         fix: bool,
     },
+    /// Clear caches based on conditions.
+    ///
+    /// A cache entry is removed if it matches any of the given conditions.
+    Evict {
+        /// Clear entries with citation keys matching this regex.
```

Comment:
Shall we provide a link to the regex specification?

Reply:
Will do!